src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodBase.cs

```diff
@@ -177,8 +177,8 @@ protected static async Task TestSelectionAsync(
             Assert.True(status.Succeeded);
         }
 
-        if (status.Succeeded && result.SelectionChanged)
-            Assert.Equal(namedSpans["r"].Single(), result.FinalSpan);
+        if (status.Succeeded && namedSpans.TryGetValue("r", out var revisedSpans))
```

Comment:
these tests were busted.  they only ran *if* the code set this boolean.  meaning if the code didn't set the oolean, the test might still specify these spans, but not validate tehm.  i've at least updated it to always validate if present.